Lombiq.JsonEditor/Views/Content_EditJsonActions.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@using OrchardCore.ContentManagement
+@using Lombiq.JsonEditor.Controllers
+
+@{
+    var contentItem = (ContentItem)Model.ContentItem;
+}
+
+<a asp-action="@nameof(AdminController.Edit)"
```

Comment:
Authorize for `CommonPermissions.EditContent` here too, so no button is displayed if you can't actually edit the item.

Reply:
Or rather in `EditJsonActionsMenuContentDisplayDriver`.

Reply:
That's even better, yes.